index.js

```diff
@@ -164,7 +165,7 @@ async function fastifyStatic (fastify, opts) {
       if (err.code === 'ENOENT') {
         // if file exists, send real file, otherwise send dir list if name match
         if (opts.list && dirList.handle(pathname, opts.list)) {
-          return dirList.send({ reply, dir: dirList.path(opts.root, pathname), options: opts.list, route: pathname })
+          return dirList.send({ request, reply, dir: dirList.path(opts.root, pathname), options: opts.list, route: pathname })
```

Comment:
There is no need to add request, it's available on `reply.request`.

Reply:
Done